microsoft-365/enterprise/microsoft-365-monitoring.md

```diff
@@ -22,7 +22,11 @@ description: "Use Microsoft 365 monitoring for information about incidents or ad
 
 # Learn about Microsoft 365 monitoring
 
-You can use dashboards in the [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=2024339) to monitor the health of various Microsoft services for your organization's Microsoft 365 subscription. This capability was initially started with Exchange Online and now getting expanded to other Microsoft services like Microsoft Teams, Microsoft 365 Apps and more service in future. Monitoring provides you with information about incidents and advisories that are collected in these categories:
+You can use dashboards in the [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=2024339) to monitor the health of various Microsoft services for your organization's Microsoft 365 subscription. This capability was initially started with Exchange Online and now getting expanded to other Microsoft services like Microsoft Teams, Microsoft 365 Apps and more service in future. 
```

Comment:
```suggestion
You can use dashboards in the [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=2024339) to monitor the health of various Microsoft services for your organization's Microsoft 365 subscription. This capability was initially started with Exchange Online and now getting expanded to other Microsoft services like Microsoft Teams, Microsoft 365 Apps, and more services in the future. 
```

Reply:
@JohanFreelancer9 Please change the first part of the last sentence to "This capability initially started with Exchange Online and now is being expanded to other Microsoft services . . . "

Reply:
Done.

Reply:
thank you for the feedback. committed with the suggestions